src/main/java/org/teamresistance/frc/Robot.java

```diff
@@ -1,13 +1,46 @@
 package org.teamresistance.frc;
 
 import edu.wpi.first.wpilibj.IterativeRobot;
+import org.strongback.Strongback;
 
 /**
  * Main robot class. Override methods from {@link IterativeRobot} to define behavior.
+ *
+ * @author Shreya Ravi
  */
 public class Robot extends IterativeRobot {
 
-    @Override
-    public void robotInit() {
-    }
+  Drive drive = drive = new Drive(IO.robotDrive, IO.leftJoystick.getRoll(), IO.leftJoystick.getPitch(), IO.rightJoystick.getRoll());
```

Comment:
oh lol idk why i have 
`Drive drive = drive = new Drive(IO.robotDrive, IO.leftJoystick.getRoll(), IO.leftJoystick.getPitch(), IO.rightJoystick.getRoll());`
 that makes no sense. I'll change that.